backend/api/appsettings.Development.json

```diff
@@ -9,6 +9,6 @@
     }
   },
   "ConnectionStrings": {
-    "PIMS": "Server=database,1433;User ID=admin;Database=pims"
+    "PIMS": "Server=database,5433;User ID=admin;Database=pims"
```

Comment:
This is incorrect and should be 1433

Reply:
Sorry, my local db is on PORT 5433. Will revert the change